README.md

```diff
@@ -202,6 +202,41 @@ connection.close();
 connector.close();
 ```
 
+### Using a Local Proxy Tunnel
+
+Another possible way to use the Cloud SQL Node.js Connector creating a
+local proxy server that creates a tunnel to the secured connection stablished
+using the `Connector.startLocalProxy()` method instead of
+`Connector.getOptions()`.
```

Comment:
```suggestion
Another possible way to use the Cloud SQL Node.js Connector is by creating a
local proxy server that tunnels to the secured connection established
using the `Connector.startLocalProxy()` method instead of
`Connector.getOptions()`.
```